lib/execution_engine2/utils/KafkaUtils.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+"""Module to provide kafka handlers for internal logging facility."""
+
+import json
+import logging
+import sys
+
+from confluent_kafka import Producer, KafkaError, Consumer
+
+_TOPICS = ["ee2"]
+
+
+def _test_sample_consumer():
```

Comment:
I would put this in a test helpers module in a separate "test" package rather than having it in here.